pages.it/common/git-help.md

```diff
@@ -0,0 +1,24 @@
+# git-help
```

Comment:
The command invoke is without hyphen, maybe we should fix this in the english page too.
```suggestion
# git help
```